src/python_testing/TC_IDM_4_2.py

```diff
@@ -409,7 +409,7 @@ async def test_TC_IDM_4_2(self):
         )
 
         # Wait MinIntervalFloor seconds before reading updated attribute value
-        time.sleep(same_min_max_interval_sec)
+        time.sleep(same_min_max_interval_sec + 0.5)
```

Comment:
Why 0.5? Why not 0.2 or 1.0? Also, this adds more time than the min interval. If this is failing, then changing the delay is NOT the solution and the test case should be revisited to be more timing-insensitive if possible.

Reply:
This does need a test plan update. @cecille working on it.

Reply:
Added loop